event_manager/lib/event_manager.rb

```diff
@@ -0,0 +1,79 @@
+require 'csv'
+require 'sunlight/congress'
+require 'erb'
+require 'date'
+
+Sunlight::Congress.api_key = "e179a6973728c4dd3fb1204283aaccb5"
```

Comment:
ok, I'll bear in mind (this code was dictated by the tutorial, however I will not do so in the future)